content/Foundation/Summit/2024/home.pml

```diff
@@ -20,11 +20,11 @@
     <li><a href="#summit-about">About</a></li>
     <!--li><a href="#summit-registration">Registration</a></li-->
     <li><a href="#summit-location">Location & Venue</a></li>
-    <!--li><a href="cfp">Call for Presentations</a></li>
-    <li><a href="#summit-keynote-speakers">Keynote Speakers</a></li>
+    <li><a class="highlight" href="cfp">Latest News: Call for Presentations</a></li>
```

Comment:
I've always found a bit strange that this does not have a section in this page. I would call this "Submission and Call For Presentation", add a section in this page with that title, and with the short text "Share your seL4 work, experience and thoughts at the seL4 summit 2024 in Sydney by submitting a presentation proposal, by the 22 of April 2024. Find all the submission information on the [Call for Presentation].